### PR TITLE
feat: Adds Control API releated Resources

### PR DESCRIPTION
**Pull request by kworkbee**
Retry of #728 due to repeated conflicts...

### CODE REVIEW

File: charts/apisix/README.md [hunk 1]
```diff
@@ -132,6 +132,25 @@ The command removes all the Kubernetes components associated with the chart and
...
+| control.ingress.annotations | object | `{}` | Ingress annotations |
+| control.ingress.hosts | list | `[{"host":"apisix-control.local","paths":["/*"]}]` | Ingress Class Name className: "nginx" |
+| control.service.annotations | object | `{}` | Control annotations |
+| control.service.externalIPs | list | `[]` | IPs for which nodes in the cluster will also accept traffic for the servic |
+| control.service.ip | string | `"127.0.0.1"` | which ip to listen on for Apache APISIX Control API |
+| control.service.port | int | `9090` | which port to use for Apache APISIX Control API |
+| control.service.servicePort | int | `9090` | Service port to use for Apache APISIX Control API |
+| control.service.type | string | `"ClusterIP"` | Control service type |
+| customPlugins | object | `{"enabled":false,"luaPath":"/opts/custom_plugins/?.lua","plugins":[{"attrs":{},"configMap":{"mounts":[{"key":"the-file-name","path":"mount-path"}],"name":"configmap-name"},"name":"plugin-name"}]}` | customPlugins allows you to mount your own HTTP plugins. |
+| customPlugins.enabled | bool | `false` | Whether to configure some custom plugins |
```
Comment (kworkbee) [inline charts/apisix/README.md:147]: I wonder why these values referred as the differences?

**Comment by kworkbee** [comment]
I tried to resolve all the conflicted issues, but it seems to be modified even if I didn't touched. Plz review all these changes.

**Comment by kworkbee** [comment]
@Revolyssup I fixed conflicts.

**Comment by kworkbee** [comment]
I'm sorry to ask you repeatedly, but please take a look.
@tao12345666333 @lingsamuel @AlinsRan @shreemaan-abhishek

**Comment by sanket-psl** [comment]
@kworkbee Any updates on this?

**Comment by kworkbee** [comment]
> @kworkbee Any updates on this?

It must be reviewed by them. Waiting for response.

**Review by Revolyssup — APPROVED** [review]

**Comment by kworkbee** [comment]
Hello @AlinsRan, PTAL

**Review by shreemaan-abhishek — APPROVED** [review]

**Comment by shreemaan-abhishek** [comment]
@kworkbee, thank you for your patience.

**Review by pottekkat — APPROVED** [review]

**Comment by brunopadz** [comment]
What's the reason to not have this PR merged yet?

**Pull request merged**